### DISCUSSION TITLE
Need a Kanban for reaching out to other Meetups

### DISCUSSION THREAD

**Discussion by misterhtmlcss** [software-developers-of-calgary/groupsite#27] [Ideas]
@venetrius I need more access please and also I need a Kanban board for organizing and speaking to all those Meetup leaders.

Please and thank you.

**Comment by venetrius** [comment]
I invited you to the project with admin role, let me know if that is what you needed. 
 @absolutelysimon I think you already reached out to several people, could you share what you learned with @misterhtmlcss ?

**Reply by misterhtmlcss** [reply]
If Simon is doing it then I'm just duplicating the work. Thank you for trying @venetrius I really appreciate it.
